File: Restaurant/Java/Item.java
package restaurant;

abstract class Item {
	String name;
	double price;
	
	Item(){
		
	}
	
	Item(String name, double price)
	{
		this.name = name;
		this.price = price;
	}
	
	abstract String getName();
	abstract double getPrice();
}

File: Restaurant/Python/Item.py
from abc import ABCMeta, abstractmethod

class Item:
	__metaclass__ = ABCMeta
	
	@abstractmethod
	def __init__(self, name, price):
		self.name = name
		self.price = price
	
	def getName(self):
		pass
		
	def getPrice(self):
		pass
		

